acs-integration-tests/src/test/java/com/ge/predix/test/utils/NewUaaTestUtil.java

```diff
@@ -0,0 +1,171 @@
+package com.ge.predix.test.utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
+import org.springframework.security.oauth2.common.exceptions.InvalidClientException;
+import org.springframework.security.oauth2.provider.client.BaseClientDetails;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import com.ge.predix.acs.utils.JsonUtils;
+
+public class NewUaaTestUtil {
+
+    private static final JsonUtils JSON_UTILS = new JsonUtils();
+
+    private final OAuth2RestTemplate adminRestTemplate;
+    private final String uaaUrl;
+    private final String tokenUrl;
+    private final ACSRestTemplateFactory acsRestTemplateFactory;
+
+    public NewUaaTestUtil(final OAuth2RestTemplate adminRestTemplate, final String uaaUrl) {
+        this.uaaUrl = uaaUrl;
+        this.tokenUrl = this.uaaUrl + "/oauth/token";
+        this.adminRestTemplate = adminRestTemplate;
+        this.acsRestTemplateFactory = null;
+    }
+
+    public NewUaaTestUtil(final ACSRestTemplateFactory acsRestTemplateFactory, final String uaaUrl) {
+        this.uaaUrl = uaaUrl;
+        this.tokenUrl = this.uaaUrl + "/oauth/token";
+        this.acsRestTemplateFactory = acsRestTemplateFactory;
+        this.adminRestTemplate = this.acsRestTemplateFactory.getOAuth2RestTemplateForUaaAdmin(this.tokenUrl);
```

Comment:
We should add TODO comments then to mark code that should be changed later when all tests are aligned.  Temporary duplicating the code is also acceptable since the duplication will go away once tests are aligned.